docs/examples/Generic Branch Example.ipynb

```diff
@@ -18,7 +18,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 7,
```

Comment:
quite minor but one of the things we do to guarantee that the example is correct, is to restart the kernel, clear all outputs and then re-running the notebook. A quick check to see whether that has happened is by checking the `execution_count`, which should start at `0` and increment by `1` for each cell.

Can you please do that?